internal/driver/driver.go

```diff
@@ -325,7 +325,7 @@ func (d *Driver) Discover() {
 		dev.Protocols[OnvifProtocol][SerialNumber] = devInfo.SerialNumber
 		dev.Protocols[OnvifProtocol][HardwareId] = devInfo.HardwareId
 		discovered := sdkModel.DiscoveredDevice{
-			Name:        fmt.Sprintf("%s-%s-%s", devInfo.Manufacturer, devInfo.Model, devInfo.SerialNumber),
+			Name:        fmt.Sprintf("%s-%s-%s", devInfo.Manufacturer, devInfo.Model, onvifDevice.GetDeviceParams().UUID),
```

Comment:
As I mentioned [in the review of the onvif library PR](https://github.com/IOTechSystems/onvif/pull/8#discussion_r813424628), we may want to handle the empty UUID case in this code.

Reply:
Modified. 
@ajcasagrande  I also make changes to the PR https://github.com/IOTechSystems/onvif/pull/8, could you help review that?